content/release-10-15-0/applications-10-15-0-bundle/10-15-0-377.md

```diff
@@ -0,0 +1,29 @@
+---
+weight: 19
+title: Release 10.15.0.377
+layout: bundle
+---
+
+<!--10.15.0.369 - 10.15.0.377-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>In the datapoint library, it is now possible to pick fragment and series from an existing datapoint.</td>

```

Comment:
what does "pick" mean exactly?
also, plural on "fragment" might work better

Reply:
I have to clarify with the team what pick means here. It is always "fragment and series". I suggest to leave this for now as Tristan is on vacation and change it later on if needed.